app/models/user.rb

```diff
@@ -0,0 +1,18 @@
+###
+#  User class for authenticating via WebAuth
+###
+class User
+  attr_accessor :webauth
+
+  def initialize(webauth)
+    @webauth = webauth
+  end
+
+  def webauth_user?
+    @webauth.present?
```

Comment:
😖 yes, that works. Sorry, I got confused and though Chris was talking about the initialize method.